### ISSUE TITLE
Laravel 10 support

### ISSUE THREAD

**Issue by Treggats** [mazedlx/laravel-feature-policy#12]
Hi, I'm going to work to get Laravel 10 support for this package. I'm going to use this issue to ask question and get early feedback.

At the moment I have these questions;

- what is the minimal PHP version you want to support?
  As Laravel 10 requires at minimum PHP 8.1
- what is the minimal Laravel version that you want to support?

I'm probably going to make a few other pull requests with additional changes, but will add questions about those here in this issue.

**Comment by Treggats** [comment]
@mazedlx what's the best / preferred way to test this in a Laravel 10 application?

Currently I have a Laravel 10 application, with my fork installed and the middleware is added to the middlewareGroups.

A `curl -i -I <site url>` returns the headers listed below.

<details>
<summary>Returned headers</summary>

```
HTTP/1.1 200 OK
Server: nginx/1.23.2
Content-Type: text/html; charset=UTF-8
Connection: keep-alive
Vary: Accept-Encoding
X-Powered-By: PHP/8.1.13
Cache-Control: no-cache, private
Date: Sat, 18 Mar 2023 14:22:25 GMT
X-Frame-Options: SAMEORIGIN
X-XSS-Protection: 1; mode=block
Referrer-Policy: strict-origin-when-cross-origin
X-Content-Type-Options: nosniff
Permissions-Policy: geolocation=self,fullscreen=self
```

</details>

**Comment by mazedlx** [comment]
There are tests in the test folder. You could make use of the Http Fake.

**Comment by mazedlx** [comment]
I've just released v2.0
